src/authn/constants.ts

```diff
@@ -1 +1 @@
-export const CCLOUD_SIGN_IN_BUTTON_LABEL: string = "Sign in to Confluent Cloud";
+export const CCLOUD_SIGN_IN_BUTTON_LABEL = "Sign in to Confluent Cloud";
```

Comment:
Main difference here is just for the dev experience and having a string vs string literal:
| |  |
|--------|--------|
| Before | <img width="404" alt="image" src="https://github.com/user-attachments/assets/49a44e03-e850-4497-8c16-2357c0c44b7e" /> | 
| After | <img width="557" alt="image" src="https://github.com/user-attachments/assets/19ec7d89-6e62-4c09-93fa-c371f9ec18d3" />
 |